tools/build_langserver/langserver/completion.go

```diff
@@ -26,25 +27,32 @@ func (h *LsHandler) handleCompletion(ctx context.Context, req *jsonrpc2.Request)
 	if err := json.Unmarshal(*req.Params, &params); err != nil {
 		return nil, err
 	}
-	log.Info("completion with param %s", req.Params)
+
 	documentURI, err := EnsureURL(params.TextDocument.URI, "file")
 	if err != nil {
+		message := fmt.Sprintf("invalid documentURI '%s' for method %s", documentURI, completionMethod)
+		log.Error(message)
 		return nil, &jsonrpc2.Error{
 			Code:    jsonrpc2.CodeInvalidParams,
-			Message: fmt.Sprintf("invalid documentURI '%s' for method %s", documentURI, completionMethod),
+			Message: fmt.Sprintf(message),
 		}
 	}
-	if !h.analyzer.IsBuildFile(documentURI) {
+	// Check if the input file is a build file or a build_defs file
+	if !h.analyzer.IsBuildFile(documentURI) || h.analyzer.IsBuildDefFile(documentURI) {
```

Comment:
Yeah I'm not sure the LPS should worry about this. Isn't it up to the client to decide what files it uses the server for?